src/Internal/Requirements/VersionValidator.php

```diff
@@ -36,7 +37,19 @@ public function validate(): bool {
 	 */
 	protected function validate_php_version() {
 		if ( ! version_compare( PHP_VERSION, '7.3', '>=' ) ) {
-			throw InvalidVersion::from_php_version( PHP_VERSION, '7.3' );
+			throw InvalidVersion::from_requirement( 'PHP', PHP_VERSION, '7.3' );
+		}
+	}
+
+	/**
+	 * Validate WordPress version being used.
+	 *
+	 * @throws InvalidVersion When the WordPress version does not meet the minimum version.
+	 */
+	protected function validate_wp_version() {
+		$wp_version = get_bloginfo( 'version' );
+		if ( ! version_compare( $wp_version, '5.5', '>=' ) ) {
```

Comment:
Can we use constants for all these minimum versions? I think it'd be easier to change later if we have them all defined in one place (possibly in the  `google-listings-and-ads.php` file where the minimum versions are  commented)